docs/Reference.md

```diff
@@ -11,14 +11,14 @@ https://discord.com/api
 ## API Versioning
 
 > danger
-> Some API and Gateway versions are now non-functioning, and are labeled as discontinued in the table below for posterity. Trying to use these versions will fail and return 400 Bad Request.
+> Some API and Gateway versions are now non-functioning and are labeled as discontinued in the table below for posterity. Trying to use these versions will fail and return 400 Bad Request.
 
-Discord exposes different versions of our API. You can specify which version to use by including it in the request path like `https://discord.com/api/v{version_number}`. Omitting the version number from the route will route requests to the current default version (marked below accordingly). You can find the change log for the newest API version [here](https://discord.com/developers/docs/change-log).
+Discord exposes different versions of our API. You can specify which version to use by including it in the request path like `https://discord.com/api/v{version_number}`. Omitting the version number from the route will route requests to the current default version (marked below accordingly). You can find the changelog for the newest API version [here](https://discord.com/developers/docs/change-log).
 
 ###### API Versions
 
 | Version | Status                           | Default |
-|---------|----------------------------------|---------|
+| ------- | -------------------------------- | ------- |
```

Comment:
looks like this commit actually changed the table formatting? the spaces aren't supposed to be in this line (was switched a while ago)

Reply:
Fixed via 5c744de1f98c29a6fe0423d8e1a4ba5793844162